src/main/java/com/jfb/ticket_app/repository/dao/UserDAO.java

```diff
@@ -21,16 +31,33 @@ public UserDAO(JdbcTemplate jdbcTemplate) {
     this.jdbcTemplate = jdbcTemplate;
   }
 
+  @Transactional
   public void saveUser(User user) {
-    jdbcTemplate.update(INSERT_USER, user.getId(), user.getName(), user.getRole(), user.getCreationTime());
+    jdbcTemplate.update(INSERT_USER, user.getId(), user.getName(), user.getRole(), user.getStatus().name(), user.getCreationTime());
   }
 
+  @Transactional
   public User getUserById(String id) {
     return jdbcTemplate.queryForObject(SELECT_USER_BY_ID, new UserRowMapper(), id);
   }
 
+  @Transactional
   public void deleteUserAndTickets(String id) {
     jdbcTemplate.update(DELETE_USER_AND_TICKETS, id);
   }
 
+  @Transactional
+  public void updateUserStatusAndCreateTicket(User user, TicketType ticketType) {
```

Comment:
add an example haw to use this method to main method

Reply:
Added an example on how to use this method.